sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -177,29 +203,38 @@ def begin_search_available_phone_numbers(
             ~azure.communication.phonenumbers.PhoneNumberAssignmentType
         :param capabilities: Required. Capabilities of a phone number.
         :type capabilities: ~azure.communication.phonenumbers.PhoneNumberCapabilities
-        :keyword str area_code: The area code of the desired phone number, e.g. 425. If not set,
+        :keyword area_code: The area code of the desired phone number, e.g. 425. If not set,
             any area code could be used in the final search.
-        :keyword int quantity: The quantity of phone numbers in the search. Default is 1.
-        :keyword str continuation_token: A continuation token to restart a poller from a saved state.
+        :paramtype area_code: str or None
+        :keyword quantity: The quantity of phone numbers in the search. Default is 1.
+        :paramtype quantity: int or None
+        :keyword continuation_token: A continuation token to restart a poller from a saved state.
+        :paramtype continuation_token: str or None
         :keyword polling: Pass in True if you'd like the LROBasePolling polling method,
          False for no polling, or your own initialized polling object for a personal polling strategy.
         :paramtype polling: bool or ~azure.core.polling.PollingMethod
-        :keyword int polling_interval: Default waiting time (seconds) between two polls
+        :keyword polling_interval: Default waiting time (seconds) between two polls
             for LRO operations if no Retry-After header is present.
+        :paramtype polling_interval: int
         :returns: A poller to wait on the search results.
         :rtype: ~azure.core.polling.LROPoller[~azure.communication.phonenumbers.PhoneNumberSearchResult]
         """
         search_request = PhoneNumberSearchRequest(
             phone_number_type=phone_number_type,
             assignment_type=assignment_type,
             capabilities=capabilities,
-            quantity=kwargs.pop("quantity", None),
-            area_code=kwargs.pop("area_code", None),
+            area_code=area_code,
         )
-        polling_interval = kwargs.pop(
-            "polling_interval", _DEFAULT_POLLING_INTERVAL_IN_SECONDS)
+        if quantity is not None:
+            search_request.quantity = quantity
```

Comment:
Is this a bug fix? We used to pass `None` in for this value if not specified.

Reply:
This was introduced because it was giving mypy errors for sending the argument as an Optional type

Reply:
But would this change result in a change of behaviour on the service?
i.e. does the service handle a `None` value the same as the value not being present?